### ISSUE TITLE
Depricated commands still available

### ISSUE THREAD

**Issue by PawelPlesniak** [DUNE-DAQ/drunc#192]
Commands that have been renamed are still present in drunc/tree/develop/src/drunc/controller.py. Trace this through with druncschema and see what is meant to be there and what is not. The commands are
 - `decsribe_fsm`
 - `get_status`
 - `execute_fsm_command`
Once done update [Running drunc](https://github.com/DUNE-DAQ/drunc/wiki/Running-drunc)

**Comment by PawelPlesniak** [comment]
Also need to add
 - `status`
 - `connect`
 - `who_am_i`
 - `fsm`
To the list of commands returned when you run `describe`

**Comment by PawelPlesniak** [comment]
Also `who_am_i` is `whoami` in the shell.
Shell defines `who-is-in-charge`, `describe` provides `who_is_in_charge`
`fsm` is not provided by `describe`
Also there is auto completion with `describe` which is probably not intended
```bash
drunc-unified-shell > describe --command describe
                                                   root-controller.test-session (controller) commands                                                    
┏━━━━━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃ name         ┃ input type ┃ return type                      ┃ help                                                                                   ┃
┡━━━━━━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
│ describe     │ None       │ request_response_pb2.Description │ Describe self (return a list of commands, the type of endpoint, the name and session). │
│ describe_fsm │ None       │ request_response_pb2.Description │ List available FSM commands for the current state.                                     │
└──────────────┴────────────┴──────────────────────────────────┴────────────────────────────────────────────────────────────────────────────────────────┘
```

TLDR - there is a lot of discrepancies between the `describe` output  and the shell command inputs. Decide on a standard.

**Comment by PawelPlesniak** [comment]
`describe` shows the endpoint commands not the shell commands, worth rewriting this to `describe_endpoint` now that `drunc` is getting used more frequently. Also may be worth writing a `process_manager` `describe` and have the unified shell run both, with a catch that pre-boot you can only see the `pm` commands not the `controller` commands

**Comment by PawelPlesniak** [comment]
When this is complete update the wiki.
